.github/workflows/build.yaml

```diff
@@ -31,8 +31,20 @@ jobs:
         strategy:
             matrix:
                 type: [gcc_debug, gcc_release, clang, mbedtls, clang_experimental]
```

Comment:
Why not just add a minimal_logging here? Then we have a single dimension and a single name for configs.

e.g. mbedtls is orthogonal to gcc/clang and debug/release and yet it is alongside them in this list..

Reply:
I could do that, yes. I initially started off with various logging options for each of the types, then scaled it way back.